src/app/public/modules/checkbox/checkbox.component.spec.ts

```diff
@@ -478,6 +478,59 @@ describe('Checkbox component', () => {
     });
   });
 
+  describe('with ngModel and an initial value', () => {
+    let checkboxElement: DebugElement;
+    let testComponent: CheckboxWithFormDirectivesComponent;
+    let inputElement: HTMLInputElement;
+    let checkboxNativeElement: HTMLElement;
+    let ngModel: NgModel;
+    let labelElement: HTMLLabelElement;
```

Comment:
Any way you can make these checks in the `describe('with ngModel')` block, instead of duplicating the assignment code in a new `describe`?

```
it('should set correct states when initialized without value', async(() => {
  fixture.someValue = undefined;
  fixture.detectChanges();
});

it('should set correct states when initialized with a value', async(() => {
  fixture.someValue = 'foobar';
  fixture.detectChanges();
});
```

Reply:
@Blackbaud-SteveBrush I had the same thought, but if you look at the `beforeEach()` for the `describe('with ngModel')` set of tests, you'll see there's a `fixture.detectChanges()` and a whole bunch of logic that follows in a `fixture.whenStable()`. I'd need the value to be set before all that happens. I figured a separate `describe` was the easier route. What do you think?

Reply:
Ugh, I guess it's okay for now. This entire spec file needs some lovin'.